openapi/components.yml

```diff
@@ -0,0 +1,38 @@
+components:
```

Comment:
This is this file changing in this PR?

Reply:
This is a small refactor whereby we're separating the api.yml into 2 files 'components.yml' and 'api.yml' and is a convention being used in other FSD repos, we could split this out into another PR if needed? :)